spec/android_localize_helper_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper.rb'
+
+describe Fastlane::Helper::Android::LocalizeHelper do
+  describe 'get_library_version_from_gradle_config' do
+    it 'returns nil when gradle file is not present' do
+      allow(File).to receive(:exists?).and_return(false)
+      expect(subject.get_library_version_from_gradle_config(import_key: 'my-test-key')).to be_nil
+    end
+
+    it 'returns nil when the key is not present' do
+      test_file_content = <<~CONTENT
+        my-test-key-foo = 'foo'
+        my-test-key-bad = 'bad'
+        my-test-key = 'my_test_value'
+      CONTENT
+
+      allow(File).to receive(:exists?).and_return(true)
+      allow(File).to receive(:open).with('./build.gradle', 'r').and_yield(StringIO.new(test_file_content))
+      expect(subject.get_library_version_from_gradle_config(import_key: 'my-test-key-b')).to be_nil
```

Comment:
```suggestion
      expect(subject.get_library_version_from_gradle_config(import_key: 'my-test-key-b')).to be_nil
      expect(subject.get_library_version_from_gradle_config(import_key: 'test-key')).to be_nil
```
I think the 2nd expectation I added here will fail (mis-catching `my-test-key`'s value) with your current code (while I expect it to pass and return `nil` instead), so worth doing some red-green-refactoring to make it work here :)

Reply:
I was torn about this tbh. I ended up adding your tests in `30846bb`, but I'm not sure the additional complexity in the regex is really worth it, because these are cases which have a really low probability in the actual code.

Reply:
I added a test to cover the prefix case in `6d8d19a`